src/components/graph.tsx

```diff
@@ -415,11 +442,12 @@ export const Graph = (props: Props) => {
     addArrowMarker("unselectedLoopArrow", "black", unselectedOpacity / lineAndLoopOpacity); // 0
 
     // draw nodes
-    const nodes = svg
-      .selectAll("g")
+    const nodes = root
+      .selectAll("g.node")
```

Comment:
Don't you want to select just `g` here since you're adding the `node` class to this element?